easycv/models/detection/utils/boxes.py

```diff
@@ -76,3 +78,332 @@ def postprocess(prediction, num_classes, conf_thre=0.7, nms_thre=0.45):
             output[i] = torch.cat((output[i], detections))
 
     return output
+
+
+def bbox2result(bboxes, labels, num_classes):
```

Comment:
does these code borrowed from mmlab? we should add declaration

Reply:
done